.mega-linter.yml

```diff
@@ -25,22 +25,24 @@ DISABLE_ERRORS_LINTERS:
   - REPOSITORY_SEMGREP
   - SPELL_VALE
 PRINT_ALL_FILES: false
-FILTER_REGEX_EXCLUDE: '(\.automation/test|\.automation/generated|\.github/workflows|docs/javascripts|docs/overrides|docs/json-schemas|flavors|clj-kondo|TEMPLATES)'
+FILTER_REGEX_EXCLUDE: '(\.automation/test|\.automation/generated|\.venv|\.github/workflows|docs/javascripts|docs/overrides|docs/json-schemas|flavors|clj-kondo|TEMPLATES)'
 JSON_JSONLINT_FILTER_REGEX_EXCLUDE: '(\.vscode/)'
 YAML_YAMLLINT_FILTER_REGEX_EXCLUDE: '(templates/\.mega-linter\.yml)'
 YAML_PRETTIER_FILTER_REGEX_EXCLUDE: '(templates/\.mega-linter\.yml|mkdocs\.yml)'
 YAML_V8R_FILTER_REGEX_EXCLUDE: '(descriptors|templates/\.mega-linter\.yml|\.codecov\.yml)'
 BASH_FILTER_REGEX_EXCLUDE: "(lib)"
 MARKDOWN_FILTER_REGEX_EXCLUDE: '(license\.md|docs/licenses)'
 MARKDOWN_MARKDOWN_LINK_CHECK_DISABLE_ERRORS: true
-SPELL_MISSPELL_FILTER_REGEX_EXCLUDE: '(\.automation/generated|docs/descriptors|used-by-stats|docs/used-by-stats\.md|.github/linters/valestyles)'
+SPELL_MISSPELL_FILTER_REGEX_EXCLUDE: '(\.automation/generated|\.venv|docs/descriptors|used-by-stats|docs/used-by-stats\.md|.github/linters/valestyles)'
```

Comment:
MISSPELL has been removed from MegaLinter v7, plz can you remove this config line ?

Reply:
done here : https://github.com/oxsecurity/megalinter/pull/2737/commits/3532da01529b0907a148f32951c7b8c1e2081d71